packages/flutter/lib/src/widgets/homogeneous_viewport.dart

```diff
@@ -79,6 +79,7 @@ abstract class _ViewportBaseElement<T extends _ViewportBase> extends RenderObjec
   void update(T newWidget) {
     bool needLayout = newWidget.isLayoutDifferentThan(widget);
     super.update(newWidget);
+    // TODO(abarth): Don't we need to update overlayPainter here?
```

Comment:
find the answer to this question before checking in. :-) (our style guide says to not check in TODOs or comments that just ask questions. :-) )